lib/bindings/HTTPBinding.js

```diff
@@ -290,6 +292,9 @@ function handleIncomingMeasure(req, res, next) {
 
         if (req.attr && req.jsonPayload) {
             config.getLogger().debug(context, 'Parsing attr [%s] with value [%s]', req.attr, req.jsonPayload);
+            if (req.body !== undefined || req.body === Object || Buffer.isBuffer(req.body)) {
+                req.jsonPayload = req.jsonPayload.toString('hex');
+            }
```

Comment:
After reviewing this PR, the hex transformation was moved to HTTPBinding file. This is not correct, since it has to be valid also for MQTT messages. 

All changes should be included in `lib/commonBindings.js`file